### PR TITLE
feat(core): Add Job Summary to Worker response

### PR DESCRIPTION
**Pull request by flipswitchingmonkey**

### CODE REVIEW

File: packages/cli/src/commands/worker.ts [hunk 1]
```diff
@@ -56,6 +56,10 @@ export class Worker extends BaseCommand {
 		[key: string]: PCancelable<IRun>;
 	} = {};
 
+	static runningJobsSummary: {
+		[key: string]: IExecutionsCurrentSummary & { name: string };
```
Comment (ivov) [inline packages/cli/src/commands/worker.ts:60]:
```suggestion
		[jobId: string]: IExecutionsCurrentSummary & { name: string };
```

Comment (ivov) [reply packages/cli/src/commands/worker.ts:60]: Let's stop using `I-` prefixes please?

File: packages/cli/src/commands/worker.ts [hunk 2]
```diff
@@ -234,11 +238,21 @@ export class Worker extends BaseCommand {
 		}
 
 		Worker.runningJobs[job.id] = workflowRun;
+		Worker.runningJobsSummary[job.id] = {
+			id: executionId,
```
Comment (ivov) [inline packages/cli/src/commands/worker.ts:242]:
```suggestion
			executionId,
```

File: packages/cli/src/commands/worker.ts [hunk 3]
```diff
@@ -234,11 +238,21 @@ export class Worker extends BaseCommand {
...
+			name: fullExecutionData.workflowData.name,
```
Comment (ivov) [inline packages/cli/src/commands/worker.ts:243]:
```suggestion
			workflowName: fullExecutionData.workflowData.name,
```

File: packages/cli/src/commands/worker.ts [hunk 4]
```diff
@@ -234,11 +238,21 @@ export class Worker extends BaseCommand {
...
+			mode: fullExecutionData.mode,
```
Comment (ivov) [inline packages/cli/src/commands/worker.ts:244]:
```suggestion
			executionMode: fullExecutionData.mode,
```

File: packages/cli/src/commands/worker.ts [hunk 5]
```diff
@@ -234,11 +238,21 @@ export class Worker extends BaseCommand {
...
+			startedAt: fullExecutionData.startedAt,
+			workflowId: fullExecutionData.workflowId ?? '',
```
Comment (ivov) [inline packages/cli/src/commands/worker.ts:246]: Can there be an execution without a workflow ID?

File: packages/cli/src/commands/worker.ts [hunk 6]
```diff
@@ -234,11 +238,21 @@ export class Worker extends BaseCommand {
...
+			retryOf: fullExecutionData.retryOf ?? '',
```
Comment (ivov) [inline packages/cli/src/commands/worker.ts:247]:
```suggestion
			retryOf: fullExecutionData.retryOf ?? null,
```

File: packages/cli/src/commands/worker.ts [hunk 7]
```diff
@@ -311,6 +325,7 @@ export class Worker extends BaseCommand {
 				instanceId: this.instanceId,
 				redisPublisher: this.redisPublisher,
 				getRunningJobIds: () => Object.keys(Worker.runningJobs),
+				getRunningJobSummary: () => Object.values(Worker.runningJobsSummary),
```
Comment (ivov) [inline packages/cli/src/commands/worker.ts:328]:
```suggestion
				getRunningJobsSummary: () => Object.values(Worker.runningJobsSummary),
```

**Review by ivov — COMMENTED** [review]
Minor things only

**Review by ivov — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.11.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.11.0)

**Pull request merged**